src/tests/ftest/soak/soak_faults.yaml

```diff
@@ -62,6 +63,7 @@ container_reserved:
     record_qty: 1
     record_size: 100
     array_size: 1
+    type: POSIX
```

Comment:
[This line](https://github.com/daos-stack/daos/blob/2af5065cec3eec0ab6615d2506be6a0bde358a60/src/tests/ftest/soak/soak.py#L906) attempts to read `object_class` from this yaml section, but I do not see it in the yaml files.  Is this intentional?